kolibri/utils/cli.py

```diff
@@ -125,15 +123,30 @@ class PluginDoesNotExist(Exception):
 
 def initialize(debug=False):
     """
-    Always called before running commands
+    Currently, always called before running commands. This may change in case
+    commands that conflict with this behavior show up.
 
     :param: debug: Tells initialization to setup logging etc.
     """
 
+    # TODO: We'll move this to a more deliberate location whereby we can
+    # ensure that some parts of kolibri can run without the whole django stack
+    django.setup()
+
     setup_logging(debug=debug)
 
     if not os.path.isfile(VERSION_FILE):
         _first_run()
+    else:
+        version = open(VERSION_FILE, "r").read()
+        if kolibri.__version__ != version.strip():
```

Comment:
This part is pretty important to avoid running `migrate` and `collectstatic` on every startup